spec/features/admin/dashboard_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe "As an admin user" do
+  describe "when I visit my admin dashboard (/admin)" do
+    it "I see all orders in the system sorted by status (packaged, pending, shipped, cancelled) and each order's id, date created, and user who placed the order which links back to the admin view of their profile" do
+      admin = create(:user, role: 3)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+
+      user_1 = create(:user, role: 0)
+      user_2 = create(:user, role: 0)
+      user_3 = create(:user, role: 0)
+
+      order_1 = create(:order, created_at: Date.today, user: user_1, status: 3) # cancelled
+      order_2 = create(:order, created_at: Date.today, user: user_2, status: 1) #packaged
+      order_3 = create(:order, created_at: Date.today, user: user_3, status: 2) # shipped
+      order_4 = create(:order, created_at: Date.today, user: user_1, status: 0) # pending
```

Comment:
The comments after order_2 and order_4 seem like they should be reversed.. 0 = packaged, 1 = pending